src/java.base/share/classes/java/lang/StackStreamFactory.java

```diff
@@ -1045,22 +1080,20 @@ private static Set<Class<?>> init() {
 
     private static boolean filterStackWalkImpl(Class<?> c) {
         return stackWalkImplClasses.contains(c) ||
-                c.getName().startsWith("java.util.stream.");
+                c.getPackageName().equals("java.util.stream");
```

Comment:
Right, that's the reason for this change.  Subpackages if added in the future may not need to be filtered as it may not the implementation of streams and method handles.